src/collection-browser.ts

```diff
@@ -155,6 +157,8 @@ export class CollectionBrowser
 
   private languageCodeHandler = new LanguageCodeHandler();
 
+  private analyticsHandler?: AnalyticsHandlerInterface;
```

Comment:
Q: Would this make sense as a reactive property rather than a private field? Does Lit allow this to be set the same way as other properties?

Reply:
yes, should be property 👍  - handy housekeeping thing to do is put `attribute: false` as lit  defaults to true @property({ type: Object, attribute: false })